scripts/postinstall.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+
+const commitlintrcPath = `${process.env.INIT_CWD}/.commitlintrc.js`;
+
+fs.open(commitlintrcPath, 'wx', err => {

```

Comment:
@linusklingzell Can't we just use `fs.statSync` if we are just checking if the file exists?

Reply:
Found `fs.existsSync` that returns a boolean, easiest implementation for our use case I think 👍